### PR TITLE
Test batch default timestamp generator

### PR DESCRIPTION
**Pull request by landim**
Need a new version of simulacron with PR: https://github.com/datastax/simulacron/pull/27

### CODE REVIEW

**Comment by landim** [comment]
Also changes the batch tests to use only 1 node cluster (instead of 3) as it wasn't verifying CL

**Comment by landim** [comment]
Rebased with update_simulacron_version branch (https://github.com/datastax/csharp-driver/pull/376)

File: src/Cassandra.IntegrationTests/Core/BatchStatements.cs [hunk 1]
```diff
@@ -210,9 +205,56 @@ public void Batch_SerialConsistency()
 
         [Test]
         [TestCassandraVersion(2, 1)]
+        public void Batch_GeneratedTimestamp()
```
Comment (jorgebay) [inline src/Cassandra.IntegrationTests/Core/BatchStatements.cs]: Great coverage using simulacron!

**Comment by jorgebay** [comment]
Included an explicit test for prepared-unprepared flow for batches and reinstated the 3-node cluster (1 node for appveyor ).

**Comment by jorgebay** [comment]
Let me know if my change looks good to you, so I can merge it.

File: src/Cassandra.IntegrationTests/Core/BatchStatements.cs [hunk 2]
```diff
@@ -28,7 +29,10 @@ public class BatchStatements : SharedClusterTest
     {
         private readonly string _tableName = "tbl" + Guid.NewGuid().ToString("N").ToLower();
 
-        public BatchStatements() : base(3)
+        /// <summary>
+        /// Use a 3-node cluster to test prepared batches (unprepared flow).
+        /// </summary>
+        public BatchStatements() : base(!IsAppVeyor ? 3 : 1)
```
Comment (landim) [inline src/Cassandra.IntegrationTests/Core/BatchStatements.cs]: Awesome!

**Pull request merged**